src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1075,8 +1095,21 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
                 }
             }
 
+            if (Form != null)
```

Comment:
Could you please use the same pattern in whole file (null on first place)? I see only 2-3 places where we use old pattern - maybe fix it too.

Reply:
fixed. Maybe comment on #5610 for the null ordering in the rest of the file (and files...)

Reply:
Done.